metadata-ingestion/tests/integration/feast/feast_repository_mces_golden.json

```diff
@@ -11,6 +11,7 @@
                 },
                 {
                     "com.linkedin.pegasus2avro.ml.metadata.MLPrimaryKeyProperties": {
+                        "customProperties": {},
```

Comment:
Does this test addition of owners + tags?

I don't see any tests for the new functionality..

Reply:
I see below you have DataHub and some tags as well..

Reply:
updated now, thanks